src/models/Player.js

```diff
@@ -7,7 +7,7 @@ const { Op } = Sequelize;
 
 class Player {
   static async getPlayerList({
-    limit, offset, order, searchPlayerName,
+    limit, offset, order, matchAll, searchPlayerName,
```

Comment:
Could you add some jsdoc for this method that explains what at the args are? `matchAll` in itself is not very descriptive.